src/Workspaces/CSharp/Portable/Formatting/Rules/TokenBasedFormattingRule.cs

```diff
@@ -201,10 +201,10 @@ public override AdjustSpacesOperation GetAdjustSpacesOperation(SyntaxToken previ
                 return CreateAdjustSpacesOperation(1, AdjustSpacesOption.ForceSpacesIfOnSingleLine);
             }
 
-            // Space for `new (` when creating a tuple array (but not a constructor constraint `where T : new()`)
+            // new (int, int)[]
             if (currentToken.Kind() == SyntaxKind.OpenParenToken &&
                 previousToken.Kind() == SyntaxKind.NewKeyword &&
-                currentToken.Parent.Kind() != SyntaxKind.ConstructorConstraint)
+                currentToken.Parent.Kind() == SyntaxKind.ObjectCreationExpression)
```

Comment:
i'm surprised you could do this on hte open paren, and didn't have to do it on the previous token (the ```new```).  I would think the openParenToken parent would be a TupleType.

Reply:
I made the change in a rush, thinking I was checking the parent of `new`. CI proves me wrong.